tracer/src/Datadog.Trace/Iast/Helpers/MongoDbHelper.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="MongoDbHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Iast.Aspects.MongoDB.DuckTyping;
+
+namespace Datadog.Trace.Iast.Helpers;
+
+internal static class MongoDbHelper
+{
+    internal static void AnalyzeBsonDocument(object command)
+    {
+        var bsonCommand = command.DuckCast<BsonDocumentCommandStruct>();
+        var jsonString = bsonCommand.Document.ToString();
+        if (string.IsNullOrEmpty(jsonString))
+        {
+            return;
+        }
+
+        DetectVulnerability(jsonString, IntegrationId.MongoDb);
```

Comment:
What I meant is that we don't need to call  DetectVulnerability(); because we dont need the result and, essentially, this method calls IastModule.OnNoSqlMongoDbQuery(); so we can just call IastModule.OnNoSqlMongoDbQuery(); directly instead of  DetectVulnerability. Anyway it's a minor thing.